packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_IPOUHarmonicCoarseOperator_def.hpp

```diff
@@ -87,7 +88,9 @@ namespace FROSch {
                                                             GOVecPtr2D dirichletBoundaryDofsVec)
     {
         FROSCH_TIMER_START_LEVELID(initializeTime,"IPOUHarmonicCoarseOperator::initialize");
+        this->dim = dimension;
```

Comment:
Why do we need the member `dim`? There is already `Dimensions_` in the `HarmonicCoarseOperator`.

Reply:
T build the repeated map on the coarse level the dimension needs to be known; we could move Dimension_ to CoarseOperator.